test/widget_test.dart

```diff
@@ -5,26 +5,24 @@
 // gestures. You can also use WidgetTester to find child widgets in the widget
 // tree, read text, and verify that the values of widget properties are correct.
 
-import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
-import 'package:math_go/app.dart';
+// import 'package:math_go/app.dart';
 
 void main() {
-  testWidgets('Counter increments smoke test', (WidgetTester tester) async {
+  testWidgets('Login flow', (WidgetTester tester) async {
     // Build our app and trigger a frame.
-    await tester.pumpWidget(MyApp());
+    // await tester.pumpWidget(MyApp());
```

Comment:
I'm gonna replace this in a separate pr since I didn't want to add the framework + tests in one pr. I left it here for reference because I don't remember how to write flutter tests 😅